qiskit/ignis/verification/tomography/data.py

```diff
@@ -98,17 +101,36 @@ def helper(x, y):
     return ret
 
 
-def count_keys(num_qubits):
-    """
-    Return ordered count keys.
+def count_keys(num_qubits: int) -> List[str]:
+    """Return ordered count keys.
+
+    Args:
+        num_qubits: The number of qubits in the generated list.
+    Returns:
+        The strings of all 0/1 combinations of the given number of qubits
+    Example:
+        >>> count_keys(3)
+        ['000', '001', '010', '011', '100', '101', '110', '111']
     """
     return [bin(j)[2:].zfill(num_qubits)
             for j in range(2 ** num_qubits)]
 
 
-def combine_counts(counts1, counts2):
-    """
-    Combine two counts dictionaries.
+def combine_counts(counts1: Dict[str, int],
+                   counts2: Dict[str, int]
+                   ) -> Dict[str, int]:
+    """Combine two counts dictionaries.
+    Args:
+        counts1: One of the count dictionaries to combine.
+        counts2: One of the count dictionaries to combine.
+    Returns:
+        A dict containing the **sum** of entries in counts1 and counts2
+        where a nonexisting entry is treated as 0
+    Example:
+        >>> counts1 = {'00': 3, '01': 5}
```

Comment:
These could be made into a `code-block`

Reply:
This is redundant for Python code, which is nicely formatted automatically when copied like this from the Python console.